tests/integration/support/subset_comparator.go

```diff
@@ -0,0 +1,196 @@
+package support
```

Comment:
Please don't copy this over. We should instead be using `go get` to fetch the dependency and use it

Reply:
It is from `facebookarchive`. Are we good to go to use that? @mikeng13

Reply:
yeah, I mentioned above we could containerize the test run so that we only install these test dependencies within that dockerfile. This can be one of them.